x-pack/plugins/security/server/config.ts

```diff
@@ -297,6 +297,11 @@ export const ConfigSchema = schema.object({
     ),
   }),
   enabled: schema.boolean({ defaultValue: true }),
+  ui: schema.object({
+    usersEnabled: schema.boolean({ defaultValue: true }),
+    rolesEnabled: schema.boolean({ defaultValue: true }),
+    roleMappingsEnabled: schema.boolean({ defaultValue: true }),
```

Comment:
Naming is hard 🙂. Can we rename these to indicate this only disables the management of these entities? I don't want to give the impression that users, roles and role mappings themselves are disabled. We're "just" disabling the management of these, but they otherwise still exist within the project.

```suggestion
    userManagementEnabled: schema.boolean({ defaultValue: true }),
    roleManagementEnabled: schema.boolean({ defaultValue: true }),
    roleMappingManagementEnabled: schema.boolean({ defaultValue: true }),
```